.github/workflows/stale.yml

```
@@ -13,7 +13,7 @@ jobs:
       pull-requests: write
     steps:
       - name: Mark/Close Stale Issues and Pull Requests
-        uses: actions/stale@v6.0.1
+        uses: actions/stale@v7.0.0
```

```suggestion
        uses: actions/stale@v7
```
@o2sh The team maintaining `actions/stale` should be force-pushing the `v7` tag to always be the latest `v7.y.z` release. Any objections to always using the latest non-breaking version instead of locking to `7.0.0`?